tests/tools/test_estmethod.py

```diff
@@ -33,30 +33,48 @@ def test_algorithm(algorithm, methods, solvers, parameter_uncertainty_methods, n
 
 
 @pytest.mark.parametrize(
-    'method, est_rec, eval_rec',
+    'method, parameter_uncertainty_method, add_eval_after_est, est_rec, eval_rec',
     [
         (
             'FO',
+            None,
+            True,
             '$ESTIMATION METHOD=ZERO INTER MAXEVAL=9999 AUTO=1 PRINT=10',
             '$ESTIMATION METHOD=IMP INTER EONLY=1 MAXEVAL=9999 ISAMPLE=10000 NITER=10 PRINT=10',
         ),
         (
             'LAPLACE',
+            None,
+            True,
```

Comment:
Since both `parameter_uncertainty_method` and `add_eval_after_est` are the same for both tests I think it's clearer to explicitly set that as the option in the tool run and not have it parametrized.

Reply:
Fair enough! The original plan was to expand the test with more cases, which explains the parameters.